<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>西邻·火车票</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/img/hack.ico</normaloff>:/img/hack.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>帐号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_username"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_password"/>
     </item>
     <item>
      <widget class="ClickedLabel" name="lb_auth_img">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>验证码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_auth_code">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginBtn">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>971</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>日志</string>
       </property>
       <widget class="QTextEdit" name="memo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>581</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>设置</string>
       </property>
       <widget class="QLabel" name="lb_speed_num">
        <property name="geometry">
         <rect>
          <x>201</x>
          <y>20</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>2s</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="group_query">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>341</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>查询</string>
        </property>
        <widget class="QLineEdit" name="edit_from_station">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="lb_to">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>到</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_to_station">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateControl">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimumDate">
          <date>
           <year>2014</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd</string>
         </property>
         <property name="date">
          <date>
           <year>2014</year>
           <month>9</month>
           <day>14</day>
          </date>
         </property>
        </widget>
        <widget class="QToolButton" name="searchBtn">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>60</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="ClickedLabel" name="lb_buy_img">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>50</y>
           <width>100</width>
           <height>40</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>dateControl</zorder>
        <zorder>edit_from_station</zorder>
        <zorder>lb_to</zorder>
        <zorder>edit_to_station</zorder>
        <zorder>searchBtn</zorder>
        <zorder>lb_buy_img</zorder>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>351</width>
          <height>351</height>
         </rect>
        </property>
        <property name="title">
         <string>预定信息</string>
        </property>
        <widget class="TableWidget" name="passager_table">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>331</width>
           <height>221</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>82</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>1</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <column>
          <property name="text">
           <string>购买</string>
          </property>
          <property name="font">
           <font>
            <family>Aharoni</family>
            <pointsize>6</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>姓名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>手机</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>身份证</string>
          </property>
         </column>
        </widget>
        <widget class="QCheckBox" name="cb_first_seat">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>250</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>一等座</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_second_seat">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>250</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>二等座</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_soft_bed">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>270</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>软卧</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_hard_bed">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>270</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>硬卧</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_hard_seat">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>290</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>硬座</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_stand">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>290</y>
           <width>91</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>无座</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="edit_train_num">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>320</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>320</y>
           <width>72</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>指定车次：</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>520</y>
          <width>261</width>
          <height>51</height>
         </rect>
        </property>
        <property name="title">
         <string>帮助</string>
        </property>
        <widget class="QPushButton" name="btn_my_blog">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>查看更新</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btn_url_12306">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>打开12036</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="lb_speed">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>22</y>
          <width>72</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>间隔：</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider">
        <property name="geometry">
         <rect>
          <x>61</x>
          <y>22</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
        <property name="tracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
       <zorder>lb_speed</zorder>
       <zorder>slider</zorder>
       <zorder>lb_speed_num</zorder>
       <zorder>group_query</zorder>
       <zorder>groupBox_4</zorder>
       <zorder>groupBox_5</zorder>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickedLabel</class>
   <extends>QLabel</extends>
   <header>clickedlabel.h</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
